diff --git a/dist/index.d.ts b/dist/index.d.ts
index e9886f1ea9b2408bf22e01d8060b1276a2ba01ec..56d1df039e1df03943d0306c8b1e823515ae5ff3 100644
--- a/dist/index.d.ts
+++ b/dist/index.d.ts
@@ -627,7 +627,7 @@ declare class Song<T = unknown> {
      * @param info             - Raw info
      * @param options          - Optional options
      */
-    constructor(info: ytdl__default.videoInfo | SearchResult | OtherSongInfo | ytdl__default.relatedVideo | RelatedSong | ytpl.result["items"][number], options?: {
+    constructor(info: ytdl__default.videoInfo | SearchResult | OtherSongInfo | ytdl__default.relatedVideo | RelatedSong | any[number], options?: {
         member?: GuildMember;
         source?: string;
         metadata?: T;
